### DISCUSSION TITLE
How to backup the configuration

### DISCUSSION THREAD

**Discussion by pippo73** [revenz/Fenrus#213] [Q&A] [answered]
Hello all, 
I've seen there is an option to install a backup, but I've not found the way to DO a backup. Could someone help me with that?
tnx all

**Comment by revenz** [comment, marked as answer]
This is only for migrating from a much older version of fenrus.   

All the configuration for Fenrus is now stored in a single database file in the data directory.  to backup your config, just backup this directory
